defi/src/protocols/data2.ts

```diff
@@ -22155,14 +22155,14 @@ const data2: Protocol[] = [
     symbol: "-",
     url: "https://www.butterswap.io/swap",
     description:
-      "Built upon the Bitcoin L2 MAP Protocol, ButterSwap is the first ZK light-client cross-chain aggregator swap that minimized ZK verification time and optimized gas fees. Exchange between popular chains & BTC ecosystem in one click!",
+      "Built upon the Bitcoin L2 MAP Protocol, ButterSwap is the first ZK light-client DEX aggregator swap that minimized ZK verification time and optimized gas fees. Exchange between popular chains & BTC ecosystem in one click!",
     chain: "Binance",
     logo: `${baseIconsUrl}/butter-swap.png`,
     audits: "0",
     audit_note: null,
     gecko_id: null,
     cmcId: null,
-    category: "Cross Chain",
+    category: "DEX Aggregator",
```

Comment:
Please remove this
Your product is cross chain
Users are able to bridge token A from chain 1 to token B on chain 2.